src/jti_background.js

```diff
@@ -44,7 +44,8 @@ if (browserType !== 'Firefox') {
 // -------------------------------------------------------------------------- //
 
 var StorageID = 'Jira-Template-Injector';
-var emptyData = {'options': {'limit': []}, 'templates': {}};
+var rateStatus = 'JTI-Rate-Status';
+var emptyData = {'options': {'limit': []}, 'templates': {}, 'status': {}};
```

Comment:
Actually you don't need to update this at all. This is only used by the export data to json functionality of the app, and we don't need to include the status's or toggles.